core/src/main/scala/cats/syntax/function1.scala

```diff
@@ -30,4 +33,47 @@ trait Function1Syntax {
      */
     def mapApply(a: A): F[B] = Functor[F].map(fab)(_(a))
   }
+
+  final class Function1FlatMapOps[F[_]: FlatMap, A, B](f: A => F[B]) {
+
+    /**
+     * Alias for `a => f(a).flatMap(g)` or `(Kleisli(f) andThen Kleisli(g)).run`
+     *
+     * Example:
+     * {{{
+     * scala> import scala.util._
+     * scala> import cats.implicits._
+     *
+     * scala> val f: List[String] => Option[String] = _.headOption
+     * scala> val g: String => Option[Int] = str => Try(str.toInt).toOption
+     * scala> (f >=> g)(List("42"))
+     * res0: Option[Int] = Some(42)
+     * scala> (f >=> g)(List("abc"))
+     * res1: Option[Int] = None
+     * scala> (f >=> g)(List())
+     * res2: Option[Int] = None
+     * }}}
+     */
+    def >=>[C](g: B => F[C]): A => F[C] = a => FlatMap[F].flatMap(f(a))(g)
```

Comment:
Should we @inline this?

Reply:
my two cents here, the other syntax ops are not inlined in general so I wouldn't do it.